src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -38,23 +38,37 @@ public static class TagHelperDescriptorFactory
         /// </summary>
         /// <param name="assemblyName">The assembly name that contains <paramref name="type"/>.</param>
         /// <param name="type">The type to create a <see cref="TagHelperDescriptor"/> from.</param>
+        /// <param name="designTime">Indicates if the returned <see cref="TagHelperDescriptor"/>s should include
+        /// design time specific information.</param>
+        /// <param name="errorSink">The <see cref="ErrorSink"/> used to collect <see cref="RazorError"/>s encountered
+        /// when creating <see cref="TagHelperDescriptor"/>s for the given <paramref name="type"/>.</param>
         /// <returns>
         /// A collection of <see cref="TagHelperDescriptor"/>s that describe the given <paramref name="type"/>.
         /// </returns>
         public static IEnumerable<TagHelperDescriptor> CreateDescriptors(
             string assemblyName,
             [NotNull] Type type,
+            bool designTime,
             [NotNull] ErrorSink errorSink)
         {
             var typeInfo = type.GetTypeInfo();
-            var attributeDescriptors = GetAttributeDescriptors(type, errorSink);
+            var attributeDescriptors = GetAttributeDescriptors(type, designTime, errorSink);
             var targetElementAttributes = GetValidTargetElementAttributes(typeInfo, errorSink);
+            TagHelperUsageDescriptor typeUsageDescriptor = null;
+
+#if !DNXCORE50
+            // The TagHelperUsageDescriptor is consumed at design time to provide IntelliSense when typing HTML
+            // elements or attributes that are tag helper related.
```

Comment:
Don't see much value in this comment or its repetition later in this file. Reads like `<remarks>` in the descriptor classes and says nothing about the line of code. Suggest removing or perhaps just saying `// TagHelperUsageDescriptor information is not needed at runtime because it used only for IntelliSense.`